base/utils/R/convert.input.R

```diff
@@ -55,6 +55,7 @@
 ##' @param forecast Logical: Is the data product a forecast?
 ##' @param ensemble An integer representing the number of ensembles, or FALSE if it data product is not an ensemble.
 ##' @param ensemble_name If convert.input is being called iteratively for each ensemble, ensemble_name contains the identifying name/number for that ensemble.
+##' @param Rbinary command (including path, if needed) to call the R executable on `host`
```

Comment:
If removing `Rbinary`, delete this line and edit line 46 above (gitHub won't let me comment on it directly because it's not yet changed in this PR) to 
```
##'   Currently only \code{host$name} and \code{host$Rbinary} are used by \code{convert.input},
##'    but whole list is passed to other functions
```